cves/CVE-2013-0863.yml

```diff
@@ -144,10 +148,11 @@ unit_tested:
     In FFmpeg, they have the FATE suite of regression tests. They are not
     "unit" tests per se, but are automated regression tests. If the fix for the
     vulnerability involved adding or updating a FATE suite, then make fix: true.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: There is a switch statement for input handling, but no formal testing.
```

Comment:
These both refer to logic, but I think the intent here is regarding FATE tests/code coverage. I'd take a quick look [here](http://fate.ffmpeg.org/) to answer this question.